packages/mint-components/src/components/sqm-referral-table/cells/sqm-referral-table-status-cell.tsx

```diff
@@ -8,10 +9,30 @@ export class ReferralTableStatusCell {
   @Prop() statusText: string;
   @Prop() converted: boolean;
   render() {
+    const style = {
```

Comment:
This can and should be moved out of the render function

Reply:
moved styles outside in `rewards/status cell`, `rewards/rewards cell` and `referral/status cell`